### DISCUSSION TITLE
Wheel filter

### DISCUSSION THREAD

**Discussion by Kaetzy** [maca88/SmartBikeLights#59] [General]
Wheel filter

Is it possible to somehow switch off the display on the navigation system?
Maybe make it dependent on the position?

When driving in the city, the display is constantly on and so is the lighting (the lighting could be turned off), but the battery runs out more quickly. If I remove the radar under the sensor everything is gone.

**Comment by maca88** [comment]
> Is it possible to somehow switch off the display on the navigation system?

Currently it is not possible the switch off the display with the data field. I don't believe that is even possible to override the Garmin setting for the display light. Try to find in the menu of your device if it is possible to disable it from there.
